python/cugraph/cugraph/tests/test_compat_algo.py

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import cugraph.experimental.compat.nx as nx
+
+
+def test_connectivity():
+    expected = list([{1, 2, 3, 4, 5}, {8, 9, 7}])
```

Comment:
Why are you converting a list that is already a list?
`list([{1, 2, 3, 4, 5}, {8, 9, 7}]) == [{1, 2, 3, 4, 5}, {8, 9, 7}]`